lib/php/libsdk/SDK/Config.php

```diff
@@ -180,7 +180,9 @@ public static function getKnownBranches() : array
 	public static function setCurrentBranchName(string $name) : void
 	{/*{{{*/
 		if (!array_key_exists($name, self::getKnownBranches())) {
-			throw new Exception("Unsupported branch '$name'");
+			fwrite(STDERR, "Unsupported branch '$name'. Falling back to 'master'.");
+
+			$name = 'master';
```

Comment:
I prefer the hard failure, instead of a soft fallback that could indicate user error.

Reply:
What would be the user error?

Reply:
Maybe for unsupported branches (and no Git available; thanks, I was missing this in my reply to #11), the user should explicitly specify the branch. If I'm not mistaken, passing `-b master` should work in this case already.